examples/http-client-endpoint/http_client_endpoint.bal

```diff
@@ -17,6 +17,9 @@ function main(string... args) {
             var msg = resp.getJsonPayload();
             match msg {
                 json jsonPayload => {
+                    // Remove headers since the user-agent will be
+                    // different from Ballerina version to version.
+                    jsonPayload.remove("headers");
```

Comment:
Instead of removing headers from the sample, try to do the modification at the test level.